tfsec-to-trivy-migration-guide.md

```diff
@@ -0,0 +1,97 @@
+# Migrating from tfsec to Trivy
+Overtime we've taken [Trivy][trivy] to be the go-to scanning tool for a vareity of things. This also includes terraform scanning.
```

Comment:
if someone lands here it's too little context. OTOH no need to repead ourselves to maybe link to  announcement instead